### DISCUSSION TITLE
SBT fails when trying to start

### DISCUSSION THREAD

**Discussion by eligolts** [sbt/sbt#6908] [Q&A]
**This is the console log. I suppose the issue came from ca-cert. Maybe somebody knows how can resolve it?**

C:/Program Files/Java/jdk1.8.0_311/bin/java.exe -Djline.terminal=jline.UnsupportedTerminal -Dsbt.log.noformat=true -Dfile.encoding=UTF-8 -Didea.managed=true -Dfile.encoding=UTF-8 -Didea.installation.dir=C:\Program Files\JetBrains\IntelliJ IDEA 2022.1 -jar C:/Users/XXX/AppData/Roaming/JetBrains/IntelliJIdea2022.1/plugins/Scala/launcher/sbt-launch.jar
[info] [launcher] getting org.fusesource.jansi jansi 1.18  (this may take some time)...
[error] [launcher] xsbt.boot.internal.shaded.coursier.error.ResolutionError$CantDownloadModule: Error downloading org.fusesource.jansi:jansi:1.18
not found: C:\Users\XXX\.ivy2\local\org.fusesource.jansi\jansi\1.18\ivys\ivy.xml
download error: Caught javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target (sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target) while downloading https://repo1.maven.org/maven2/org/fusesource/jansi/jansi/1.18/jansi-1.18.pom
download error: Caught javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target (sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target) while downloading https://repo.scala-sbt.org/scalasbt/maven-releases/org/fusesource/jansi/jansi/1.18/jansi-1.18.pom
download error: Caught javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target (sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target) while downloading https://repo.scala-sbt.org/scalasbt/maven-snapshots/org/fusesource/jansi/jansi/1.18/jansi-1.18.pom
download error: Caught javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target (sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target) while downloading https://repo.typesafe.com/typesafe/ivy-releases/org.fusesource.jansi/jansi/1.18/ivys/ivy.xml
download error: Caught javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target (sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target) while downloading https://repo.scala-sbt.org/scalasbt/ivy-snapshots/org.fusesource.jansi/jansi/1.18/ivys/ivy.xml
[error] [launcher] could not retrieve jansi 1.18

**Comment by SethTisue** [comment]
almost certainly the problem has nothing to do with sbt per se, but is just a JVM issue. so whatever solutions you find by googling the error message, without including "sbt" in the search, should be applicable

is there a need for you to be on Java 8? you might be able to spare yourself some trouble by using 11 or 17 instead
